client/src/components/exercise/lab12/Main.js

```diff
@@ -26,15 +26,16 @@ const Main = () => {
     <div className="bottomSpace">
       <GameStateContext.Provider value={{ exerciseState, setExerciseState }}>
         <Router className="app">
-          <Test path="/*" />
+          <ExerciseIntro path="/" />
           <Diploma path="/diploma" />
           <GradApplication path="/GraduationApplication" />
-          <ExerciseIntro path="/ExerciseIntro" />
           <PreWrongDiploma path="/PreWrongDiploma" />
           <PostWrongNewsletter path="/PostWrongNewsletter" />
           <PreCorrectDiploma path="/PreCorrectDiploma" />
           <PostCorrectNewsletter path="/PostCorrectNewsletter" />
           <KeyTakeaways path="/KeyTakeaways" />
+          {/* TODO: name needs to come from db */}
+          <AlumniNewsletter path="/AlumniNewsletter" name="Test" />
```

Comment:
Should use the current user's name using `state.main.user` and get the data from there. Or if someone isn't logged in, make a section where a user can enter their name prior to starting the lab here?

Reply:
The name will come from the form that users fill out before they say the diploma and the alumni newsletter (both the wrong name and their preferred name). Rather than `state.main.user`, I think the name should come from the context we're using to capture the form input